ostd/src/mm/page_prop.rs

```diff
@@ -17,14 +17,28 @@ pub struct PageProperty {
 }
 
 impl PageProperty {
+    /// Creates a new `PageProperty` with the given flags, cache policy and privilege flags.
```

Comment:
```suggestion
    /// Creates a new `PageProperty` with the given flags, cache policy, and privilege flags.
```

However, if you address the comments below, I think this method will become dead and disappear.

Code inside OSTD tends to explicitly construct the `PageProperty`, like:
https://github.com/asterinas/asterinas/blob/1ed023f4136c5b7132c12f3dfa4c62198f17bed4/ostd/src/mm/kspace/mod.rs#L145-L149

Reply:
Thank you for pointing out. I just noticed that there are many explicit construction of `PageProperty` like this in OSTD. I think the method you proposed (including that in the comment above) should be better than this PR.

What do you think of this? @junyang-zh I guess I'll follow the suggestions here if you're also OK with it.

Reply:
I am OK with the new approach, that is to remove the `new` method.